internals/src/Tizen.Inspections/Tizen.Inspections/DataRequestReceivedEventArgs.cs

```diff
@@ -0,0 +1,50 @@
+/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
```

Comment:
I suggest changing the year if it is a new file.
```suggestion
* Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
```